pkg/controllers/dynakube/controller_test.go

```diff
@@ -392,7 +392,7 @@ func TestReconcileComponents(t *testing.T) {
 
 		require.Error(t, err)
 		// goerrors.Join concats errors with \n
-		assert.Len(t, strings.Split(err.Error(), "\n"), 7) // ActiveGate, Extension, OtelC, OneAgent LogMonitoring, and Injection reconcilers
+		assert.Len(t, strings.Split(err.Error(), "\n"), 7) // ActiveGate, Extension, OtelC, OneAgent LogMonitoring, Injection and OTLP reconcilers
```

Comment:
should be KSPM, not OTLP. how about we make each reconciler return a unique error string and check that this aggregated error contains all of the expected strings instead of having this comment?

Reply:
fixed the comment in [b31611b](https://github.com/Dynatrace/dynatrace-operator/pull/5491/commits/b31611be1f33f2e39461c799263d37a94c5b91e5). Adapting the error strings should be done in a seperate PR though, as this is out of scope, and the PR is already quite large